frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+{
+  "appName": "ऐप फ्लोवी",
+  "defaultUsername": "में",
+  "welcomeText": "स्वागत है @:appName",
+  "githubStarText": "गिटहब पैर स्टार दे",
+  "subscribeNewsletterText": "हमारे न्यूज़ की जानकारी पाए",
+  "letsGoButtonText": "शुरू करे",
+  "title": "टाइटल",
+  "signUp": {
+    "buttonText": "साईन करे",
+    "title": "साईन करे @:appName",
+    "getStartedText": "शुरू करे",
+    "emptyPasswordError": "पासवर्ड खाली नहीं रख सकते",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है",
+    "alreadyHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "repeatPasswordHint": "रिपीट पासवर्ड"
+  },
+  "signIn": {
+    "loginTitle": "लॉगिन करे @:appName",
+    "loginButtonText": "लॉगिन",
+    "loginAsGuestButtonText": "शुरू करे",
+    "buttonText": "साईन करे",
+    "forgotPassword": "क्या आप पासवर्ड भूल गए ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "dontHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है"
+  },
+  "workspace": {
+    "create": "काम करने की जगह बनाये",
+    "hint": "काम करने की जगह",
+    "notFoundError": "काम करने की जगह नहीं मिली"
+  },
+  "shareAction": {
+    "buttonText": "शेयर करे",
+    "workInProgress": "जल्द आ रहा है",
+    "markdown": "मार्क कीजिये",
```

Comment:
markdown should stay `मार्कडाउन` because in this case, it is a noun and not an action, मार्क कीजिये refers to the act of marking something, which is not what markdown means here.